From 75d718988a62eb9540fa7b117f81634ae4fe42d4 Mon Sep 17 00:00:00 2001
From: Salome Thirot <salome.thirot@arm.com>
Date: Wed, 9 Oct 2024 11:58:00 +0100
Subject: [PATCH 02/56] Remove 12-bit test cases for highbd_warp_affine

SVT-AV1 doesn't support 12-bit encoding, so no need to test these.
---
 test/warp_filter_test_util.cc | 7 +------
 1 file changed, 1 insertion(+), 6 deletions(-)

diff --git a/test/warp_filter_test_util.cc b/test/warp_filter_test_util.cc
index 0d180b7f..a2d3242a 100644
--- a/test/warp_filter_test_util.cc
+++ b/test/warp_filter_test_util.cc
@@ -404,11 +404,6 @@ namespace AV1HighbdWarpFilter {
         make_tuple(64, 64, 100, 10, filter),
         make_tuple(4, 16, 100, 10, filter),
         make_tuple(32, 8, 100, 10, filter),
-        make_tuple(4, 4, 100, 12, filter),
-        make_tuple(8, 8, 100, 12, filter),
-        make_tuple(64, 64, 100, 12, filter),
-        make_tuple(4, 16, 100, 12, filter),
-        make_tuple(32, 8, 100, 12, filter),
     };
     return ::testing::Combine(::testing::ValuesIn(params),
                               ::testing::Values(0, 1),
@@ -420,7 +415,7 @@ namespace AV1HighbdWarpFilter {
 AV1HighbdWarpFilterTest::~AV1HighbdWarpFilterTest() {
 }
 void AV1HighbdWarpFilterTest::SetUp() {
-    rnd_ = new svt_av1_test_tool::SVTRandom(0, (1 << 12) - 1);
+    rnd_ = new svt_av1_test_tool::SVTRandom(0, (1 << 10) - 1);
 }
 
 void AV1HighbdWarpFilterTest::TearDown() {
-- 
2.36.0.windows.1

